torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8224,11 +8238,12 @@ def __call__(self, opinfo, device, dtype, requires_grad, **kwargs):
         _foreach_inputs_kwargs = {k: kwargs.pop(k, v) for k, v in _foreach_inputs_default_kwargs.items()}
 
         for num_tensors, ord in product(num_input_tensors, (0, 1, 2, -1, -2)):
-            input = sample_inputs_foreach(None, device, dtype, num_tensors, **_foreach_inputs_kwargs)
-            disable_fastpath = True
-            if ord in (1, 2) and dtype in floating_types_and(torch.half, torch.bfloat16):
-                disable_fastpath = False
-            yield SampleInput(input, **{"ord": ord, "disable_fastpath": disable_fastpath})
+            for zero_size in (False,):
```

Comment:
why?

Reply:
I was a bit too lazy so redundantly skip zero_size cases. `ord` of -1 & -2 seem to be skipped better as `RuntimeError: linalg.vector_norm cannot compute the -2 norm on an empty tensor because the operation does not have an identity`